libvast/src/system/exporter.cpp

```diff
@@ -292,6 +298,7 @@ behavior exporter(stateful_actor<exporter_state>* self, expression expr,
           VAST_IGNORE_UNUSED(e);
           VAST_DEBUG(self, "failed to lookup query at index:",
                      self->system().render(e));
+          shutdown(self, e);
```

Comment:
It looks like the error gets now rendered twice: in the above `VAST_DEBUG` statement as well as in the new `shutdown` overload. I think one time suffices.